scripts/helpers.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------------------------------------------------------------------------------------------
+local helpers = {}
+-----------------------------------------------------------------------------------------------------------------------
+function helpers.get_sdk_version()
+    local v = os.getWindowsRegistry("HKLM:SOFTWARE\\Wow6432Node\\Microsoft\\Microsoft SDKs\\Windows\\v10.0\\ProductVersion") .. ".0"
+    return v
+end
+-----------------------------------------------------------------------------------------------------------------------
+helpers.sfml = {}
+helpers.box2d = {}
+helpers.wxwidgets = {}
+
+helpers.sfml.root = os.getenv("SFML_ROOT")
```

Comment:
would be good to have some .bat template which set those and then runs project generation